components/InlineVideo.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <video
+    style="clip-path: inset(2px 2px)"
+    width="100%"
+    height="100%"
+    controls
+  >
+    <source
+      src="~/assets/webauthn-demo.mp4"
```

Comment:
Can we pass this in as a prop so that we can make this a re-usable component?

Reply:
I tried this a few different ways but I can't seem to get it working without some errors.

Reply:
ah, gotcha, we can figure it out at a later time then.